test/TokenBalanceOracle.js

```diff
@@ -223,64 +173,6 @@ contract('TokenBalanceOracle', ([appManager, accountBal900, accountBal100, accou
-          await acl.createPermission(appManager, executionTarget.address, SET_COUNTER_ROLE, appManager)
-          await acl.grantPermissionP(accountBal900, executionTarget.address, SET_COUNTER_ROLE, params)
-          await acl.grantPermissionP(accountBal100, executionTarget.address, SET_COUNTER_ROLE, params)
-          await acl.grantPermissionP(accountBal0, executionTarget.address, SET_COUNTER_ROLE, params)
-        })
-
-        context(`Required balance is ${ORACLE_MINIMUM_BALANCE}`, () => {
-          it('can set counter if account has more than minimum required balance', async () => {
-            const expectedCounter = 3
-
-            await executionTarget.setCounter(expectedCounter, { from: accountBal900 })
-
-            const actualCounter = await executionTarget.counter()
-            assert.equal(actualCounter, expectedCounter)
-          })
-
-          it(`can set counter if account has exactly the minimum required balance`, async () => {
-            await executionTarget.setCounter(1, { from: accountBal100 })
-          })
-
-          it("can't set counter if account does not have tokens", async () => {
-            await assertRevert(executionTarget.setCounter(1, { from: accountBal0 }), 'APP_AUTH_FAILED')
-          })
-        })
-
-        context('Required balance is 1', () => {
-          beforeEach('set minimum required balance to 1', async () => {
-            await acl.createPermission(appManager, oracle.address, SET_MIN_BALANCE_ROLE, appManager)
-            await oracle.setMinBalance(1)
-          })
-
-          it('all accounts with positive balance can set counter', async () => {
-            await executionTarget.setCounter(1, { from: accountBal900 })
-            await executionTarget.setCounter(1, { from: accountBal100 })
-          })
-
-          it("can't set counter if account does not have tokens", async () => {
-            await assertRevert(executionTarget.setCounter(1, { from: accountBal0 }), 'APP_AUTH_FAILED')
-          })
-        })
-
-        context(`required balance is ${MOCK_TOKEN_BALANCE * 2}`, () => {
-          beforeEach(`set minimum required balance to ${MOCK_TOKEN_BALANCE * 2}`, async () => {
-            await acl.createPermission(appManager, oracle.address, SET_MIN_BALANCE_ROLE, appManager)
-            await oracle.setMinBalance(MOCK_TOKEN_BALANCE * 2)
-          })
-
-          it('no account can set counter', async () => {
-            await assertRevert(executionTarget.setCounter(1, { from: accountBal900 }), 'APP_AUTH_FAILED')
-            await assertRevert(executionTarget.setCounter(1, { from: accountBal100 }), 'APP_AUTH_FAILED')
-            await assertRevert(executionTarget.setCounter(1, { from: accountBal0 }), 'APP_AUTH_FAILED')
-          })
-        })
-      })
-
       //address as auth param
       describe('executing function with address auth param', () => {
         beforeEach('Create role and grant with params', async () => {
```

Comment:
The calls in this `beforeEach`  can go in the previous `beforeEach` and this and the above `describe('executing...` can be removed.